lib/drush-node.js

```diff
@@ -65,40 +68,78 @@ Drush.version = function () {
  *
  * @return Promise
  */
-Drush.exec = function (command, options) {
+Drush.exec = function (args, options) {
   options = options || {};
+  if (typeof args === 'string') {
+    args = args.split(' ');
```

Comment:
Might want to use a library, like [yargs](https://www.npmjs.com/package/yargs) to parse this out since '-a    -b' and '-a	-b` and `-a -b` are all equivalent but would render different results here.